src/main/java/com/google/api/gax/grpc/ApiCallable.java

```diff
@@ -51,16 +50,89 @@
 
 /**
  * A callable is an object which represents one or more rpc calls. Various operators on callables
- * produce new callables, representing common API programming patterns. Callables can be used to
- * directly operate against an api, or to efficiently implement wrappers for apis which add
+ * produce new callables, representing common API programming patterns. Callables can make API
+ * method calls directly, or to efficiently implement wrappers for apis which add
  * additional functionality and processing.
  */
 @ExperimentalApi
 public class ApiCallable<RequestT, ResponseT> {
+
   private final FutureCallable<RequestT, ResponseT> callable;
 
+  @Nullable
+  private final ApiCallSettings settings;
+
+  /**
+   * Create a callable object that represents a simple API method.
```

Comment:
Document on all of these create methods that they are public only for technical reasons; they are for advanced usage (generated client wrappers)